sdk/translation/azure-ai-translation-text/tests/test_translation.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under the MIT License.
 # ------------------------------------
 
+import os
```

Comment:
Nit: I dont see where os was added?

Reply:
Removed, it was added by one of my attempts for test